chart/kubeapps/values.yaml

```diff
@@ -198,6 +198,7 @@ frontend:
   ## Some K8s distributions such as GKE requires it
   ##
   proxypassAccessTokenAsBearer: false
+  proxypassExtraSetHeader: Authorization "Bearer $cookie_sessionid";
```

Comment:
Sweet, that'll be a nice addition for the chart. Ooh, and that will make it trivial to do "anonymous" access demos (as in, the user can deploy kubeapps with a pre-configured bearer token of a service-account so no login is required for demos on throwaway clusters (obviously demo only, very dangerous for anything exposed).